static/app/components/events/autofix/autofixSteps.tsx

```diff
@@ -456,3 +451,57 @@ const ProgressContainer = styled('div')`
 const ProgressStepContainer = styled('div')`
   grid-column: 1/-1;
 `;
+
+function LogComponent({html}) {
```

Comment:
This component should have a type for its props

Reply:
Done in next commit